pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: mime
-version: 1.0.2
+version: 1.0.3
```

Comment:
This can be kept as `1.0.2`. I apparently haven't published that yet and we may as well roll this out with it.

Reply:
d1

Moved change to in `CHANGELOG` to `1.0.2`